include/imu_ros2/acceleration_ros_publisher.h

```diff
@@ -27,6 +27,10 @@
 #include <rclcpp/rclcpp.hpp>
 #include <sensor_msgs/msg/imu.hpp>
 
+#define SETTING_MODE 0
```

Comment:
I think configuration mode instead of setting mode would be better (see for all publishers)